modules/distribution/resources/operation_policies/specifications/ccAddHeader_v2.json

```diff
@@ -0,0 +1,35 @@
+{
+  "category": "Mediation",
+  "version": "v2",
+  "displayName": "Add Header",
+  "name": "ccAddHeader",
```

Comment:
we can use apk instead of cc, also in file names as well

Reply:
Fixed in [6c11e76](https://github.com/wso2/product-apim/pull/13498/commits/6c11e765736c5f60336c667d95ce3e7aa39982f7)